microsite/data/plugins/tech-updates.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Tech Updates
+author: Infosys
+authorUrl: https://www.infosys.com
+category: Utility
+description: A tech news updates is to keep users informed about the latest updates.
+documentation: https://github.com/AmbrishRamachandiran/tech-updates/blob/main/README.md
+iconUrl: /img/tech-updates-logo.png
+npmPackageName: '@backstage/plugin-tech-updates'

```

Comment:
You need to publish your package to NPM, you won't be able to use the `@backstage` organization. This for sure will need to be fixed so that others can pull down your package and use it 👍